src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSegmentEntry.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RegressionGames;
+using RegressionGames.StateRecorder;
+using RegressionGames.StateRecorder.BotSegments;
+using RegressionGames.StateRecorder.BotSegments.Models;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+/**
+ * <summary>
+ * Displays the high-level information for a Segment, and enables the user to play it
+ * </summary>
+ */
+public class RGSegmentEntry : MonoBehaviour
+{
+    public string segmentName;
+
+    public string description;
+    
+    // will be a file path when in-editor, or a resources path when in a packaged build
+    public string path;

```

Comment:
keep this a 'filePath' and 'resourcePath' like the sequences are

we will need that information later for properly indicating to users which segments are loading from a local override vs in the build

Reply:
good call. These 2 have been separated